p2p/stream/common/streammanager/cooldown.go

```diff
@@ -0,0 +1,40 @@
+package streammanager
+
+import (
+	"container/list"
+	"time"
+
+	"github.com/libp2p/go-libp2p-core/peer"
+
+	"github.com/whyrusleeping/timecache"
+)
+
+const (
+	coolDownPeriod = 1 * time.Minute
```

Comment:
Can you add more explanation about the reason for adding the cooldown period and why it's 1 min?

Reply:
Thanks. Added comments.